src/AutoRest.CSharp/Common/AutoRest/Plugins/CSharpProj.cs

```diff
@@ -32,8 +32,12 @@ internal class CSharpProj : IPlugin
 ";
 
         private string _armCsProjContent = @"
+  <PropertyGroup>
+    <IncludeManagementSharedCode>true</IncludeManagementSharedCode>
```

Comment:
Who would set this for azure-sdk-for-net?

Reply:
in azure-sdk-for-net its automatically included if you are a mgmt track 2 library [here](https://github.com/Azure/azure-sdk-for-net/blob/feature/mgmt-track2/eng/Directory.Build.Common.targets#L146)